data/esau-morais.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "Esaú Morais",
+  "displayStatsPublic": false,
+  "type": "personal",
+  "bio": "Full-Stack Web Developer",
+  "avatar": "https://avatars.githubusercontent.com/u/55207584?v=4",
```

It's best to use this link as it's shorter and updates automatically.
```suggestion
  "avatar": "https://github.com/esau-morais.png",
```

---

This is more dynamic and will use their latest profile picture from GitHub but I don't think it is a blocker

---

Oh it was updated, great 👍